Documentation/Concepts/Extensions/Index.rst

```diff
@@ -7,9 +7,92 @@
 Extensions
 ==========
 
-.. figure:: /Images/Illustrations/extensions.png
-
 Extensions are pieces of software developed by the TYPO3 community that extend
-the functionality of a TYPO3 installation. Extensions come in many forms
-- from small extensions that carry out specific tasks to larger extensions that
-provide an entire suite of functionality, e.g. the TYPO3 Blog Extension.
+the functionality of a TYPO3 installation. Extensions come in many forms.
+Some are only used for one site and contain mainly the theme of that site.
```

Comment:
Maybe say something like "other systems often call this plugins or add-ons". So that SEO can find this here?

Also maybe first give example about functions? Like:

"Extensions can provide things like:
- the theme/layout of a site (such a variant is called "sitepackage" specifically), also offering content element definitions
- Frontend Functionality like: Search, Logins, News-Articles, Blogs, "CRUD" applications to manage record data, Communities, Workflow implementations, ...
- Backend modules
- Configuration presets
- API/SDK integrations
- Third-Party integrations like Matomo, Cookie Banners, Schema.org API

Reply:
I feel like this would be too much information in the very first contact with this topic. We mention more of this where we introduce some examples of third party extensions and tell ppl how to find extensions

Reply:
I suggest adding an info box here with some info about terminology. This would cover @fe-hicking's point about "other systems often call this plugins or add-ons." People transitioning from other CMSs *will* have a problem with terminology.

A TYPO3 "extension" is a Drupal "module" is a WordPress "plugin". And both "module" and "plugin" have specific meanings in the TYPO3 universe.

Reply:
There is a section complete with what a plugin is in TYPO3 and what it is compared to a plugin below. we can make a follow up with such an info box or some more information about. I currently have no real idea

Reply:
Yes. I'll make a separate PR with info box content on this topic once this PR is merged.